NitroxServer/ConsoleCommands/SayCommand.cs

```diff
@@ -3,22 +3,28 @@
 using NitroxModel.Packets;
 using NitroxServer.ConsoleCommands.Abstract;
 using NitroxServer.GameLogic;
+using NitroxModel.DataStructures.GameLogic;
 
 namespace NitroxServer.ConsoleCommands
 {
     internal class SayCommand : Command
     {
         private readonly PlayerManager playerManager;
 
-        public SayCommand(PlayerManager playerManager) : base("say", Optional<string>.Of("<message>"), "say Even the lowliest of cogs needs to say something SO SAY SOMETHING!", new[] {"broadcast"})
+        public SayCommand(PlayerManager playerManager) : base("say", NitroxModel.DataStructures.GameLogic.Perms.Admin, Optional<string>.Of("<message>"), "say Even the lowliest of cogs needs to say something SO SAY SOMETHING!", new[] {"broadcast"})
         {
             this.playerManager = playerManager;
+            SupportsClientSide = true;
```

Comment:
Are some not supported client side?

Reply:
used the SupportsClientSide to differentiate between commands like exit and say, there isnt a need to tell the client about the exit command so we dont have to differentiate between the Player execution method and the server execution method